adc_raw:
  acquisition_rate: 100